src/libs/SearchUIUtils.ts

```diff
@@ -356,11 +371,28 @@ function getAction(data: OnyxTypes.SearchResults['data'], key: string): SearchTr
  */
 function getReportActionsSections(data: OnyxTypes.SearchResults['data']): ReportActionListItemType[] {
     const reportActionItems: ReportActionListItemType[] = [];
+
+    const transactions = Object.keys(data)
+        .filter((key): key is TransactionKey => isTransactionEntry(key))
```

Comment:
`isTransactionEntry` is already a type guard, so why are we wrapping this in another typeguard? 
Can you try a more simple:
```
.filter(isTransactionEntry)
```
?

Reply:
same comment for all 3 calls